### ISSUE TITLE
 Can't create layer "317" of type "Clip" 

### ISSUE THREAD

**Issue by wait1988** [opencv/opencv#15043]
##### System information (version)
- OpenCV == 4.1
- Operating System / Platform => Windows 64 Bit
- Compiler => Visual Studio 2015

classify.onnx is mobilenetv2 model exported from pytorch,you can use the torchvison version.
Net net = readNetFromONNX("classify.onnx");
Then get the error:
opencv-4.1.0\modules\dnn\src\dnn.cpp:525: error: (-2:Unspecified error) Can't create layer "317" of type "Clip" in function 'cv::dnn::dnn4_v20190122::LayerData::getLayerInstance'

**Comment by dkurt** [comment]
Must be fixed in https://github.com/opencv/opencv/pull/14959 by @dvd42. Let me close this issue. Thanks!

**Comment by wait1988** [comment]
@dkurt The Clip op is supported,but the other error occurs:
Can't create layer "464" of type "ReduceMean" in function 'cv::dnn::experimental_dnn_34_v13::LayerData::getLayerInstance'

some onnx outputs:
 %462 : Float(1, 1280, 7, 7) = onnx::BatchNormalization[epsilon=1e-05, momentum=0.9](%461, %features.18.1.weight, %features.18.1.bias, %features.18.1.running_mean, %features.18.1.running_var), scope: MobileNetV2/Sequential[features]/ConvBNReLU[18]/BatchNorm2d[1]
  %463 : Float(1, 1280, 7, 7) = onnx::Clip[max=6, min=0](%462), scope: MobileNetV2/Sequential[features]/ConvBNReLU[18]/ReLU6[2]
  %464 : Float(1, 1280, 7) = onnx::ReduceMean[axes=[3], keepdims=0](%463), scope: MobileNetV2
  %465 : Float(1, 1280) = onnx::ReduceMean[axes=[2], keepdims=0](%464), scope: MobileNetV2/Sequential[classifier]/Dropout[0]
  %466 : Float(1, 8) = onnx::Gemm[alpha=1, beta=1, transB=1](%465, %classifier.1.weight, %classifier.1.bias), scope: MobileNetV2/Sequential[classifier]/Dropout[0]

**Comment by wait1988** [comment]
Maybe this pr can solve the problem [https://github.com/opencv/opencv/pull/15032#issue-297003816](url)

**Comment by dkurt** [comment]
@wait1988, there is already opened issue about it: https://github.com/opencv/opencv/issues/14768. You may track it.
